lessons/lesson2.md

```diff
@@ -1,129 +1,102 @@
-## Lesson 2: Write your app
+## Lesson 2: Register the App as Event Provider 
 
-# H1
-## H2
-### H3
-#### H4
-##### H5
-###### H6
+### Create a Console Integration and Set up Project
+In order to use custom event SDK, you need to get below informtion from console integration.
 
-Emphasis, aka italics, with *asterisks* or _underscores_.
+- `IMS Org Id`: The Organization Id in which the provider, event metadata, etc are to be created. 
+- `API key`: The API Key ( client id ) for the integration ( project workspace ) 
+- `JWT Token`: Note that I/O Management Service needs to be enabled for the integration
+- `Config.zip`: Config file downloaded from console including private key and certificate_pub.crt
+- `project.json`: for instance, `projectname-orgId-Production.json` file downloaded from console 
 
-Strong emphasis, aka bold, with **asterisks** or __underscores__.
 
-Combined emphasis with **asterisks and _underscores_**.
+1. Navigate to Adobe I/O console at [https://console.adobe.io](https://console.adobe.io) in your browser and create a project or using your exsiting project 
+2. Select `Add to Project` -> `Add an API` -> `Adobe Services` -> `I/O managemenet API`
+![add-api](assets/add-api.png)
 
-Strikethrough uses two tildes. ~~Scratch this.~~
+3. Follow the steps to configure API, create a new service account (JWT) credential, `config.zip` will be
+downloaded automatically, you will need the private key to generate JWT token.
 
-1. First ordered list item
-2. Another item
-⋅⋅* Unordered sub-list. 
-1. Actual numbers don't matter, just that it's a number
-⋅⋅1. Ordered sub-list
-4. And another item.
+4. Go to `project overview` tab, download project metadata from below `Download` button and get the needed info from this `.json`file, or you can also get these info from `.aio` file in the project folder.
```

Comment:
are credentials in the screenshots faked? secrets should not be shown publicly.

Reply:
done

Reply:
In fact with the new CLI, you don't need these screenshots... All credentials are managed by the CLI.
Better to delete them all so that credentials are not publicly exposed.